addon/mixins/course/rollover-controller.js

```diff
@@ -1,9 +1,9 @@
 import Mixin from '@ember/object/mixin';
+import { action } from '@ember/object';
 
 export default Mixin.create({
-  actions: {
-    loadCourse(newCourse){
-      this.transitionToRoute('course', newCourse);
-    }
+  @action
```

Comment:
I think this makes this a breaking change, noting it to remember what needs to get updated on frontend.

Reply:
Actually I'm not sure this is valid syntax, I think decorators are only valid on classes.

Reply:
i'm looking into it....

Reply:
Click-tested through the frontend. this hold up.

Reply:
I thought the templates on the frontend would have to be updated. Ok. Great!